### DISCUSSION TITLE
How to Use with W5500 ETH chip

### DISCUSSION THREAD

**Discussion by Wolly300** [arendst/Tasmota#22720] [Q&A] [answered]
Hello together,

I'm using this ESP32-S3 Board with POE.

https://www.waveshare.com/esp32-s3-eth.htm

![image](https://github.com/user-attachments/assets/f538d191-2a67-474d-bbe4-4a8aaeb3ee0e)

I used the web installer to install the latest (14.4.1 en) version of Tasmota and hooked up the ESP to my POE switch. The ESP did not connect via ETH, so I configured the Wi-Fi and changed 4 pins for the SPI, but still no ETH connection.

![image](https://github.com/user-attachments/assets/0f588665-dd3c-409d-9d2d-05f53c324f1f)

I checked out the release notes, and it says "Support W5500 SPI ethernet using four SPI GPIOs only without IRQ and RESET", the support came with 14.1.0, so I guess it should work.

Where is the problem, what did I forget to configure to get the ETH connection?

**Comment by Wolly300** [comment]
*
I tried to flash the tasmota32s3.bin version of the 14.4.1, but I got image block errors after the restart.

**Comment by Wolly300** [comment]
Got it working, forgot executing the EthType 8 command

**Reply by Noschvie** [reply]
How did you get it to work?

**Reply by Wolly300** [reply]
To get the ETH is pretty simple.

First, set up the pin configuration.
<img width="437" alt="image" src="https://github.com/user-attachments/assets/fdc87767-5378-408f-8cad-4bde8eb63378" />

 * SPI_MOSI
 * SPI_MISO
 * SPI_CLK
 * SPI_RST = Tasmota ETH_POWER
 * SPI_IRQ = Tasmota ETH_MDIO
 * SPI_CS  = Tasmota ETH_MDC

Then enter this command **EthType 8** in the console.

**Reply by belveder79** [reply]
@Wolly300 Can you please comment if it also works without PoE? It seems that Ethernet only works with PoE but does not work without?

**Reply by Wolly300** [reply]
> @Wolly300 Can you please comment if it also works without PoE? It seems that Ethernet only works with PoE but does not work without?

To be honest, I never tried it with a regular switch port, just PoE ones. But I can just recommend using ESPHome instead. It works way better and is far better to set up, maintain, and debug.

**Reply by belveder79** [reply]
Ok, I got mine and I tested it. Works fine with both, Wifi, PoE and standard Ethernet. Thx for the config...

**Reply by ortegafernando** [reply]
> Ok, I got mine and I tested it. Works fine with both, Wifi, PoE and standard Ethernet. Thx for the config...

Great, could you please share with your experience? @belveder79

**Reply by belveder79** [reply]
Well, my experience with the board is pretty limited. I got the one with the PoE but without the camera. So I can't really say if attaching a camera and also using some dumping to the SD-Card works with the current 14.4.1. 

I'm running it since yesterday continuously including some berry scripting and it works like every other ESP32 board. Overall I like it, but the PoE functionality comes at a price tag that is not negligible. So my current assessment is that I'd get this board only if I desperately need PoE or want to stream camera images over Ethernet (to be tested some time later). If I was about just doing regular ESP32 stuff or poor-mans webcam stuff, I'd pick a normal Dev Board or the ESPCam...

**Comment by Wolly300** [comment]
I also got the ESP32-S3 version running.

I'm using the ESP to read out my energy meter via SML, so I had to recompile it with the SML functions.

1. Open the current master branch in gitpod ([https://gitpod.io#https://github.com/arendst/Tasmota/tree/master](https://gitpod.io/#https://github.com/arendst/Tasmota/tree/master)
2. Edit the user_config_override.h
3. Compile with this command: platformio run -e tasmota32s3
4. Download the .bin
5. Use ESP Flasher to flash the ESP32 (https://github.com/Jason2866/ESP_Flasher/releases)

**Comment by ortegafernando** [comment]
Hi there is any TASMOTA functionallity to use an ESP32 with a W5500 as a wifi access point? Thanks

**Reply by sfromis** [reply]
While Tasmota does have the [Range Extender](https://tasmota.github.io/docs/Range-Extender/) feature to bridge traffic to an AP (working for things like using a tablet to have Internet access via that AP), I think that this is implemented with the assumption that the base network is also Wifi, not wired.

I've not verified this limitation, but the linked docs text does suggest "routed to the core WiFi network".

**Reply by ortegafernando** [reply]
Ok, I will try an write here about it. If I get managed I will also try to make a PR of documentation.

**Comment by tobias-friedrich** [comment]
I installed the standard **[tasmota32s3.bin](https://ota.tasmota.com/tasmota32/)** (14.5.0.2) on my **[Waveshare ESP32-S3-ETH](https://www.waveshare.com/esp32-s3-eth.htm)**, powered via USB-C (no PoE installed). Wi-Fi works fine.

Inspired by @Wolly300, I’ve set the following [configuration](https://tasmota.github.io/docs/Commands/#ethernet):
- **Ethernet settings:**
  - `EthType 8`
  - `EthClockMode 3`
- **Template:**
  ```json
  {
    "NAME": "Waveshare ESP32-S3-ETH",
    "GPIO": [0,0,0,0,640,0,0,0,608,3872,7648,704,672,736,768,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
    "FLAG": 0,
    "BASE": 1
  }

Unfortunately, I still can’t get the onboard W5500 Ethernet chip to work.

#### Questions:
1. If Ethernet works for anyone using the standard **[tasmota32s3.bin](https://ota.tasmota.com/tasmota32/)**, could you please share your template?
2. If compiling custom firmware is necessary, what settings did you use in [user_config_override.h](https://github.com/arendst/Tasmota/blob/master/tasmota/my_user_config.h)?

#### Thanks!!

**Reply by s-hadinger** [reply]
I have never tried on esp32s3. Can you please share the boot logs ideally with serial attached and set `seriallog 4` or `weblog 4`

**Reply by tobias-friedrich** [reply]
Sure, here's the `weblog 4`:


```
00:00:00.253 CMD: Fall back to serial port, no SOF packet detected on USB port
00:00:00.254 HDW: ESP32-S3 v0.2 
00:00:00.269 UFS: FlashFS mounted with 12592 kB free
00:00:00.282 CFG: Loaded from File, Count 47
00:00:00.291 QPC: Count 1
00:00:00.292 CFG: CR 412/699, Busy 0
00:00:00.298 SPI: Bus1 using GPIO13(CLK), GPIO11(MOSI) and GPIO12(MISO)
00:00:00.300 I2C: Bus1 using GPIO08(SCL) and GPIO04(SDA)
00:00:00.306 TFS: File 'mcp23x.dat' not found
00:00:00.306 MCP: Invalid template
00:00:00.307 TYA: Active=0
00:00:00.309 ROT: Mode 1
00:00:00.317 BRY: GC from 3692 to 3492 bytes, objects freed 4/20 (in 0 ms) - slots from 36/61 to 28/61
00:00:00.337 CFG: No '*.autoconf' file found
00:00:00.342 BRY: GC from 5327 to 3796 bytes, objects freed 8/35 (in 1 ms) - slots from 54/76 to 35/76
00:00:00.343 BRY: Berry initialized, RAM used 3796 bytes
00:00:00.351 BRY: No 'preinit.be'
00:00:00.355 SRC: Restart
00:00:00.360 Project tasmota - Tasmota Version 14.5.0.2(e3eb470-tasmota32)-3_1_3(2025-03-08T14:23:45)
00:00:00.361 ETH: No ETH MDC as SPI CS GPIO defined
00:00:00.374 SHT: Sensor did not ACK command
00:00:00.421 BRY: No 'autoexec.be'
00:00:00.503 WIF: Checking connection...
00:00:00.503 WIF: Attempting connection...
00:00:01.001 WIF: Connecting to AP1 FF Maschine Channel 1 BSSId 7A:83:C2:14:BD:48 in mode HT40 as tasmota-FB67DC-2012...
00:00:01.124 WIF: IPv4 192.168.22.182, mask 255.255.192.0, gateway 192.168.0.1
00:00:02.813 WIF: Checking connection...
00:00:02.814 WIF: Connected
00:00:03.024 HTP: Web server active on tasmota-FB67DC-2012 with IP address 192.168.22.182
00:00:03.414 WIF: IPv6 Local fe80::a285:e3ff:fefb:67dc%st1
00:00:03.722 NTP: Sync time...
00:00:03.789 WIF: DNS resolved '2.pool.ntp.org' (78.47.168.188) in 66 ms
00:00:03.967 RTC: UTC 2025-03-09T09:33:10Z, DST 2025-03-30T02:00:00, STD 2025-10-26T03:00:00
10:33:10.000 RTC: Synced by NTP
10:33:10.031 RSL: INFO1 = {"Info1":{"Module":"Waveshare ESP32-S3-ETH","Version":"14.5.0.2(e3eb470-tasmota32)","FallbackTopic":"cmnd/DVES_FB67DC_fb/","GroupTopic":"cmnd/tasmotas/"}}
10:33:10.045 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-FB67DC-2012","IPAddress":"192.168.22.182","IP6Global":"","IP6Local":"fe80::a285:e3ff:fefb:67dc%st1"}}
10:33:10.059 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":21}}
10:33:10.075 TFS: File 'autoexec.bat' not found
10:33:12.722 QPC: Reset
10:33:13.414 WIF: IPv6 Global fd07:f253:b95f:6047:a285:e3ff:fefb:67dc
10:33:14.745 APP: Boot Count 21
10:33:14.747 BRY: GC from 5285 to 4108 bytes, objects freed 18/45 (in 1 ms) - slots from 80/91 to 49/91
10:33:14.750 RSL: STATE = {"Time":"2025-03-09T10:33:14","Uptime":"0T00:00:09","UptimeSec":9,"Heap":225,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":4,"Objects":45},"Wifi":{"AP":1,"SSId":"FF Maschine","BSSId":"7A:83:C2:14:BD:48","Channel":1,"Mode":"HT40","RSSI":42,"Signal":-79,"LinkCount":1,"Downtime":"0T00:00:04"}}
10:33:15.502 HTP: Main Menu
10:33:15.920 CFG: Saved, Count 48, Bytes 4096
10:33:19.704 RSL: config = {"ip":"192.168.22.182","dn":"Tasmota","fn":["Tasmota",null,null,null,null,null,null,null],"hn":"tasmota-FB67DC-2012","mac":"A085E3FB67DC","md":"Waveshare ESP32-S3-ETH","ty":0,"if":0,"cam":0,"ofln":"Offline","onln":"Online","state":["OFF","ON","TOGGLE","HOLD"],"sw":"14.5.0.2","t":"tasmota_FB67DC","ft":"%prefix%/%topic%/","tp":["cmnd","stat","tele"],"rl":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"swc":[-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1],"swn":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"btn":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"so":{"4":0,"11":0,"13":0,"17":0,"20":0,"30":0,"68":0,"73":0,"82":0,"114":0,"117":0},"lk":0,"lt_st":0,"bat":0,"dslp":0,"sho":[],"sht":[],"ver":1}
10:33:19.710 RSL: sensors = {"sn":{"Time":"2025-03-09T10:33:19"},"ver":1}
10:33:29.439 WIF: Checking connection...
10:33:41.009 HTP: Tools
10:33:42.282 HTP: Console
10:33:49.442 WIF: Checking connection...

```

And here's the 'seriallog 4':

```
10:46:57.416 HTP: Restart
10:46:57.418 SRC: WebGui from 192.168.50.218
10:46:57.814 WIF: Checking connection...
10:46:58.573 APP: Restarting
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0xc (RTC_SW_CPU_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
Saved PC:0x4037caf6
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce2820,len:0x1ac
load:0x403c8700,len:0x4
load:0x403c8704,len:0xb30
load:0x403cb700,len:0x2890
SHA-256 comparison failed:
Calculated: f31443e36cb6acfa3825fbe74533af2518818acd46ce45f11e3f0e067e02e7f0
Expected: 7044964501eea83b74459c9ac6eacb4a778b5cd9a5733fdb103ec9ae3b187bbf
Attempting to boot anyway...
entry 0x403c8868

00:00:00.001 CMD: Using USB CDC
00:00:00.002 HDW: ESP32-S3 v0.2 
00:00:00.014 UFS: FlashFS mounted with 12592 kB free
00:00:00.024 CFG: Loaded from File, Count 53
00:00:00.025 SER: Set to 8N1 115200 bit/s
00:00:00.025 SER: HWCDC supports 115200 bit/s only
00:00:00.033 QPC: Count 1
00:00:00.034 CFG: CR 412/699, Busy 0
00:00:00.040 SPI: Bus1 using GPIO13(CLK), GPIO11(MOSI) and GPIO12(MISO)
00:00:00.043 I2C: Bus1 using GPIO08(SCL) and GPIO04(SDA)
00:00:00.048 TFS: File 'mcp23x.dat' not found
00:00:00.049 MCP: Invalid template
00:00:00.050 TYA: Active=0
00:00:00.052 ROT: Mode 1
00:00:00.060 BRY: GC from 3692 to 3492 bytes, objects freed 4/20 (in 0 ms) - slots from 36/61 to 28/61
00:00:00.077 CFG: No '*.autoconf' file found
00:00:00.083 BRY: GC from 5327 to 3796 bytes, objects freed 8/35 (in 1 ms) - slots from 54/76 to 35/76
00:00:00.084 BRY: Berry initialized, RAM used 3796 bytes
00:00:00.091 BRY: No 'preinit.be'
00:00:00.095 SRC: Restart
00:00:00.100 Project tasmota - Tasmota Version 14.5.0.2(e3eb470-tasmota32)-3_1_3(2025-03-08T14:23:45)
00:00:00.102 ETH: No ETH MDC as SPI CS GPIO defined
00:00:00.115 SHT: Sensor did not ACK command
00:00:00.157 BRY: No 'autoexec.be'
00:00:00.244 WIF: Checking connection...
00:00:00.245 WIF: Attempting connection...
00:00:01.001 WIF: Connecting to AP1 FF Maschine Channel 1 BSSId 7A:83:C2:14:BD:48 in mode HT40 as tasmota-FB67DC-2012...
00:00:01.108 WIF: IPv4 192.168.22.182, mask 255.255.192.0, gateway 192.168.0.1
00:00:02.836 WIF: Checking connection...
00:00:02.837 WIF: Connected
00:00:02.990 NTP: Sync time...
00:00:03.035 WIF: DNS resolved '2.pool.ntp.org' (158.180.28.150) in 42 ms
00:00:03.263 RTC: UTC 2025-03-09T09:47:02Z, DST 2025-03-30T02:00:00, STD 2025-10-26T03:00:00
10:47:02.000 RTC: Synced by NTP
10:47:02.024 HTP: Web server active on tasmota-FB67DC-2012 with IP address 192.168.22.182
10:47:02.149 WIF: IPv6 Local fe80::a285:e3ff:fefb:67dc%st1
10:47:03.093 RSL: INFO1 = {"Info1":{"Module":"Waveshare ESP32-S3-ETH","Version":"14.5.0.2(e3eb470-tasmota32)","FallbackTopic":"cmnd/DVES_FB67DC_fb/","GroupTopic":"cmnd/tasmotas/"}}
10:47:03.097 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-FB67DC-2012","IPAddress":"192.168.22.182","IP6Global":"","IP6Local":"fe80::a285:e3ff:fefb:67dc%st1"}}
10:47:03.100 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":26}}
10:47:03.105 TFS: File 'autoexec.bat' not found
10:47:05.977 QPC: Reset
10:47:06.412 WIF: IPv6 Global fd07:f253:b95f:6047:a285:e3ff:fefb:67dc
10:47:07.997 HTP: Main Menu
10:47:08.184 APP: Boot Count 26
10:47:08.188 BRY: GC from 5285 to 4108 bytes, objects freed 18/45 (in 1 ms) - slots from 80/91 to 49/91
10:47:08.194 RSL: STATE = {"Time":"2025-03-09T10:47:08","Uptime":"0T00:00:10","UptimeSec":10,"Heap":224,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":4,"Objects":45},"Wifi":{"AP":1,"SSId":"FF Maschine","BSSId":"7A:83:C2:14:BD:48","Channel":1,"Mode":"HT40","RSSI":38,"Signal":-81,"LinkCount":1,"Downtime":"0T00:00:03"}}
10:47:08.743 CFG: Saved, Count 54, Bytes 4096
10:47:12.991 RSL: config = {"ip":"192.168.22.182","dn":"Tasmota","fn":["Tasmota",null,null,null,null,null,null,null],"hn":"tasmota-FB67DC-2012","mac":"A085E3FB67DC","md":"Waveshare ESP32-S3-ETH","ty":0,"if":0,"cam":0,"ofln":"Offline","onln":"Online","state":["OFF","ON","TOGGLE","HOLD"],"sw":"14.5.0.2","t":"tasmota_FB67DC","ft":"%prefix%/%topic%/","tp":["cmnd","stat","tele"],"rl":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"swc":[-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1],"swn":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"btn":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"so":{"4":0,"11":0,"13":0,"17":0,"20":0,"30":0,"68":0,"73":0,"82":0,"114":0,"117":0},"lk":0,"lt_st":0,"bat":0,"dslp":0,"sho":[],"sht":[],"ver":1}
10:47:13.001 RSL: sensors = {"sn":{"Time":"2025-03-09T10:47:12"},"ver":1}
10:47:22.119 WIF: Checking connection...
10:47:42.123 WIF: Checking connection...
10:48:02.178 WIF: Checking connection...
10:48:22.131 WIF: Checking connection...
```

**Reply by s-hadinger** [reply]
The message is quite explicit:
```
ETH: No ETH MDC as SPI CS GPIO defined
```

**Reply by s-hadinger** [reply]
Did you set `module 0` to enable the template?

**Reply by tobias-friedrich** [reply]
Well spotted! I didn't see the `ETH: No ETH MDC as SPI CS GPIO defined` in the log. Setting GPIO14 as SPI CS was inspired by @Wolly300. Also Waveshare denotes GPIO14 as ETH_CS. So I guess it's indeed the wrong template.

I enabled this template:
<img width="255" alt="image" src="https://github.com/user-attachments/assets/0c4c9abb-6999-4267-828c-21de3434f844" />

**Reply by s-hadinger** [reply]
So does it work now?

**Reply by tobias-friedrich** [reply]
Unfortunately, no. The screenshot of the template is the same as the one I posted before, and it appears to be incorrect. I may need to set ETH MDC and ETH MDIO, but I haven’t been able to determine which GPIOs should be used.

**Comment by Noschvie** [comment]
Have a look to [Discussions 18207](https://github.com/arendst/Tasmota/discussions/18207)

have the board running with and without PoE and will share the template this evening if needed.

**Reply by Noschvie** [reply, marked as answer]
These settings are working 
<img width="355" alt="image" src="https://github.com/user-attachments/assets/5f9821fa-78bc-4bb6-a17b-29fc2f29a970" />

**Reply by tobias-friedrich** [reply]
Beautiful! Thanks, @Noschvie! Now ethernet works like charm. 😃

**Comment by Brodel012** [comment]
Hallo,
habe mir das Board gekauft weil ich es sehr cool finde gerade noch mit POE.

Ich habe es jetzt mit dem web Install Tasmota versucht in gang zu setzten nur ohne erfolg kein WLAN und keine Netzwerk verbingung.
Ohne WLAN kommt man ja garnicht erst zum einstellen der LAN Schnitstelle.

Ich habejetzt eine WLAN Antenne angesteckt, habe es damit versucht.
Auch habe ich verschidene .bin dateien versucht nur leider ohne erfolg. 

tasmota32s3.bin https://ota.tasmota.com/tasmota32/tasmota32s3.bin   1997k	20250629 23:19

ESP Startet nach dem flchen nicht und es kommt auch nicht wie üblich mit wlan verbinden nach der installation.

Kann mir da jemand beiter helfen?

**Reply by Noschvie** [reply]
"das Board" ist welches genau?
Verwende den [Tasmota Online Installer](https://tasmota.github.io/install/), der erkennt automatisch die Hardware und wählt das passende Firmware File.

**Reply by Brodel012** [reply]
Waveshare ESP32-S3 ETH Development Board

okay ich habe immer eins Ausgewählt. Hat funktioniert vielen Dank.

Netztwerk bekomme ich nur noch nicht hin, da scheint noch was nicht zu stimmen.
Wenn ich ein Template anlege, wird es nicht übernommen. Selbst bei wenn ich es direkt bei den modulen einstelle.

**Reply by Noschvie** [reply]
Hast du beim Template "Activate" enabled ?

**Reply by Noschvie** [reply]
Console Command: `EthType 8` notwendig, anschliessend kann das WIFI mit `wifi 0` disabled werden.

**Reply by Brodel012** [reply]
Vielen Dank
Ja einschalten hat geholfen mit EthType 8 

jetzt muss ich es nur noch auf Deutsch bekommen.

**Reply by Noschvie** [reply]
Um ein deutsches UserInterface zu bekommen, ist ein "self-compiled" Image für den ESP32-S3 zu erstellen. Rate dir jedoch davon ab, da du dies dann für jede neue Version zu erstellen hast.
Das Tasmota GUI ist nicht für den Alltag gedacht...

**Reply by Brodel012** [reply]
Vielen Dank 
da nehme ich mir den Rat an und lass es so, sehr gute hilfe.

**Reply by Noschvie** [reply]
You're welcome!

**Reply by Gifford47** [reply]
Has anyone managed to initialise the SD card successfully?

**Comment by YM-Tsang** [comment]
how to set manual_ip: with w5500?

**Reply by sfromis** [reply]
The Ethernet-related commands are in the docs, including `EthIPAddress`
https://tasmota.github.io/docs/Commands/#ethernet

**Comment by larrykluger** [comment]
I've added the ESP32-S3-ETH and -POE-ETH board-specific config info to the docs. 

Please upvote or comment on the [Pull Request](https://github.com/tasmota/docs/pull/1513) 

thank you!

**Comment by RN-BIT** [comment]
That was very helpful. Now I'd like to set up the camera. How should I proceed?

**Reply by sfromis** [reply]
Better not ask offtopic questions in a thread dedicated to a very different topic. You can search for example camera configs in https://templates.blakadder.com/

**Comment by goofy79** [comment]
Hello, have the ESP32-S3 Ethernet Dev Board with PoE - but no way to get the eth work.
Actual i use the [Tasmota 15.3.0 (release-tasmota32) by Theo Arends] (tasmota32s3.bin) - can you tell me what i can try ? 

Thank you so much

**Reply by sfromis** [reply]
Details depend on what your board is (saying "ESP32-S3 Ethernet Dev Board with PoE" does not identify the specific board), but for me the Waveshare ESP32-S3-ETH board with W5500 (already talked about in this thread) works with these gpios:
```
Gpio.09  AO 5536 ETH POWER
Gpio.10  AO 5600 ETH MDIO
Gpio.11  AO 704  SPI MOSI1
Gpio.12  AO 672  SPI MISO1
Gpio.13  AO 736  SPI CLK1
Gpio.14  AO 5568 ETH MDC1
```
And config settings: EthType 8; EthAddress 0; EthClockMode 3
Should be similar for other boards, but you need to be sure of what gpios are used.
